frontend/src/components/organisms/ViewEvents.tsx

```diff
@@ -13,34 +13,35 @@ import Link from "next/link";
 type Action = "rsvp" | "cancel rsvp" | "publish" | "manage attendees" | "edit";
 
 const UpcomingEvents = () => {
-  return (
-    <CardList>
-      <EventCard
-        eventid={"000"}
-        mainAction={"rsvp"}
-        dropdownActions={["cancel rsvp"]}
-        title={"snehar"}
-        location={"jameson"}
-        datetime={"today lol"}
-      />
-      <EventCard
-        eventid={"000"}
-        mainAction={"rsvp"}
-        dropdownActions={["cancel rsvp"]}
-        title={"snehar"}
-        location={"jameson"}
-        datetime={"today lol"}
-      />
-      <EventCard
-        eventid={"000"}
-        mainAction={"rsvp"}
-        dropdownActions={["cancel rsvp"]}
-        title={"snehar"}
-        location={"jameson"}
-        datetime={"today lol"}
-      />
-    </CardList>
-  );
+  return <></>;
+  // return (
```

Comment:
Try not to merge this into main while commented out (remove, change, keep, etc...) 🙏

Reply:
Was commented out for deployment purposes but will be uncommented once a different PR is merged (which fixes event cards)